packages/api/src/store/table.ts

```diff
@@ -332,24 +332,24 @@ export default class Table<T extends DBObject> {
 
   // on startup: auto-create indices if they don't exist
   async ensureIndex(propName, prop) {
-    if (!prop.index && !prop.unique) {
-      return;
-    }
-    let unique = "";
-    if (prop.unique) {
-      unique = "unique";
-    }
-    const indexName = `${this.name}_${propName}`;
-    try {
-      await this.db.query(`
-          CREATE ${unique} INDEX "${indexName}" ON "${this.name}" USING BTREE ((data->>'${propName}'));
-        `);
-    } catch (e) {
-      if (!e.message.includes("already exists")) {
-        throw e;
-      }
-      return;
-    }
-    logger.info(`Created ${unique} index ${indexName} on ${this.name}`);
+    // if (!prop.index && !prop.unique) {
```

Comment:
So this only means that we will not belê creating new indexes right? Or did this ensure here cause performance issues even when the indexes were already there? Or did you have to delete some existing indices by any chance and can't have them recreated?